components/Cms/Element/CmsElementImageSlider.vue

```diff
@@ -1,53 +1,79 @@
 <script setup lang="ts">
+
 const props = defineProps<{
     element: CmsElementImageSlider;
 }>();
+// Main slider html object
+const sliderRef = ref(null);
 
 const config = useCmsElementConfig(props.element);
-const sliderItems = config.getConfigValue('sliderItems');
+const sliderItems = computed(() => props.element.data?.sliderItems ?? []);
+const slides = ref(Array.from({ length: sliderItems.value.length }));
+
+if (sliderItems.value.length > 0) {
+    // Main slider swiper instance
+    useSwiper(sliderRef, {
+
+    });
+}
+const autoplayConfig = computed(( )=>{
```

Comment:
```suggestion
const autoplayConfig = computed(() => {
```
did you tun `bun run lint`?